### ISSUE TITLE
Add "stars added in the date range" in github card

### ISSUE THREAD

**Issue by siva-kranthi** [medyo/hackertab.dev#172]
It would be good if you add `starsInDateRange` in GitHub card

**Comment by medyo** [comment]
Could you develop more your idea, if you can suggest a mockup tool would be great

**Comment by siva-kranthi** [comment]
Like devo extension I want to see the **number of stars added** based on the today/week/month selection eg: **255 starts today**. It would be very helpful in identifying the popular repos

![image](https://github.com/medyo/hackertab.dev/assets/558871/d3ff3209-775f-41d9-80d6-774f22035f62)

**Comment by medyo** [comment]
@siva-kranthi yes we can do that, the info is already available in the api (`starsInDateRange :int`), would you like to work on it?
